packages/devextreme/js/__internal/ui/chat/chat_message_list.ts

```diff
@@ -79,8 +100,30 @@ class MessageList extends Widget<MessageListOptions> {
         this._createMessageGroupComponent(currentMessageGroupItems, currentMessageGroupUserId);
       }
     });
+  }
+
+  _renderMessage(message: Message, newItems: Message[], sender: User): void {
+    this._setOptionWithoutOptionChange('items', newItems);
```

Comment:
I don't understand why we should use set option without option change. Could you please clarify reasons?

Reply:
You and I discussed this last week. Nothing has changed: we decided that if we change the entire message array, we need to call _invalidate to rerender all message groups

However, in this case, when only one new message is added, we don't want to rerender everything, we draw the message separately and update the items in silent mode

Reply:
Yes, this is true for the parent component, but it's unclear why this synchronization is needed here. Alright, let's leave it as is until the next iterations.